.github/workflows/check-pr.yml

```diff
@@ -26,3 +29,148 @@ jobs:
         run: npm ci
       - name: Run validation and tests
         run: npm pack
+
+      - name: Trigger Workflows
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.WORKFLOW_TRIGGER_SECRET }}  # or use a custom token with proper permissions
+          repository: rdkcentral/firebolt-apis 
+          event-type: trigger-workflow
+          client-payload: '{"OPENRPC_PR_BRANCH": "${{ github.event.pull_request.head.ref }}"}'
+
+      - name: Store the workflow run ID
```

Comment:
I don't see any logic where we're storing a run ID.

Reply:
I removed this,added mistakenly actually workId is getting from the reponse of Curl request https://github.com/rdkcentral/firebolt-openrpc/blob/92c5402185ddbad240e82fe842d7ce34e2f770ac/.github/workflows/check-pr.yml#L60